### PR TITLE
Geocover query

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: src/query/QueryManager.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      if (popupContent) {
+        popupContent = popupContent.replace(/cell-left/g, 'key')
+        .replace(/<td>/g, '<td class="value">')
+        .replace(/<table>/g, '<table class="ui compact small very basic table">');
+      }
+      const onshow = () => {
+        console.log('showing', d.geometry);
+      };
+      const onhide = () => {
+        console.log('hiding', d.geometry);
```
Comment (vladyslav-tk) [inline src/query/QueryManager.js:35]: Is console.log required?

File: src/query/QueryManager.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      };
+      return {
+        popupContent,
+        onhide,
+        onshow
+      };
+    }
+  }
+
+  async onclick(click) {
```
Comment (vladyslav-tk) [inline src/query/QueryManager.js:45]: There are some issues with a click:

1. When clicking out of map it returns the error: https://i.imgur.com/wtJvdN6.jpeg
2. When drawing popup appears: https://i.imgur.com/dY8oiTW.jpeg

Comment (gberaudo) [reply src/query/QueryManager.js:39]: I fixed these, thanks.

Comment (vladyslav-tk) [reply src/query/QueryManager.js:38]: You disabled query when tools are open and now AOI cant be selected when AOI section is open

**Pull request merged**